amr-wind/wind_energy/ABLBoundaryPlane.cpp

```diff
@@ -186,19 +271,32 @@ ABLBoundaryPlane::ABLBoundaryPlane(CFDSim& sim)
         return;
     }
 
-#ifndef AMR_WIND_USE_NETCDF
-    if (m_is_initialized) {
-        amrex::Abort(
-            "ABLBoundaryPlane capability with IO mode requires NetCDF");
-    }
-#else
     pp.query("bndry_write_frequency", m_write_frequency);
     pp.queryarr("bndry_planes", m_planes);
     pp.query("bndry_output_start_time", m_out_start_time);
     pp.queryarr("bndry_var_names", m_var_names);
     pp.get("bndry_file", m_filename);
+    pp.query("bndry_output_format", m_out_fmt);
 
+#ifndef AMR_WIND_USE_NETCDF
+    if (m_out_fmt == "netcdf") {
+        amrex::Print()
+            << "Error: boundary output format using netcdf must link netcdf "
```

Comment:
I would expect an "error" to quit the program. Can we change to "warning"?

Reply:
good call, I'll change it